assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

```diff
@@ -168,3 +168,7 @@ che.keycloak.username_claim=NULL
 # If set to "delegated", then the service will use Keycloak IdentityProvider mechanism.
 # Runtime Exception wii be thrown, in case if this property is not set properly.
 che.oauth.service_mode=delegated
+
+# Configuration for enabling removing user from Keycloak server on removing user from Che database.
+# By default should be false, will need only in some special cases
+che.keycloak.cascade_user_removal_enabled=true
```

Comment:
```suggestion
che.keycloak.cascade_user_removal_enabled=false
```

Reply:
Thanks, for catching

Reply:
In which special cases it's going to be needed?

Reply:
I think it makes sense to add references to credentials here as well, like java class has:
```
For correct work need to set admin username ${che.keycloak.admin_username} and password
 * ${che.keycloak.admin_password}.
```